tools/gui/qtkontakt.py

```diff
@@ -294,7 +294,7 @@ def __lade_alle_werte(self):
                     return
             
             # Wird nur bei Terminsuche benötigt
-            self.i_code_impfzentren.setText(kontaktdaten["code"])
+            self.i_code_impfzentren.setText(kontaktdaten["codes"][0])
```

Comment:
es ist nicht ganz korrekt immer den ersten Code aus der Liste anzuzeigen, oder?

Reply:
also das behebt vllt schon das Problem, aber nicht die Ursache

Reply:
Ja, dazu muss auch die Speicherfunktion angepasst werden. Habe ich bei mir lokal schon gemacht.
Ohne zu Wissen, wie wir das in der GUI darstellen wollen (Es geht gerade nur ein einzelner Code), würde ich das aber als Hotfix ersteinmal anwenden.

Reply:
In der anderen Datei hatte ich das so angezeigt: https://github.com/iamnotturner/vaccipy/pull/351/files#diff-00babc323e04b1bf84252abb3c98fc91894ccf71c6fba6a04de96657c689211bR143

Aber das kann man so natürlich nicht abspeichern, bzw. erst dann, wenn man die Speicherfunktion anpasst, damit sie eine kommaseparierte Liste an Codes unterstützt.